### ISSUE TITLE
not working 

### ISSUE THREAD

**Issue by nik1977** [Vaneixus/rpi-gadget-image#1]
Hi,
I'm using pi zero w 1.1 with the prebuild image and nothing is happening only LED is blinking 7 times and pausing. I'm not sure if i have done things right.I have no experience with linux nor github. May be I'm missing some additional configurations.

**Comment by Vaneixus** [comment]
May I ask how you have written the image to the sd card? and which led is blinking. oh and I am currently working on a new image that I have tested just a few seconds ago, if you give me 2 hours, I can get you a working build. I think one of the earlier ones is broken, I think this one is one of the few of the actually working old builds in the list: https://github.com/Vaneixus/rpi-gadget-image/actions/runs/6477408704
otherwise, the new build I am working on is for adding new functionality. which is why if you downloaded the last green checkmarked build, it might have been an experimental image. The one currently built *should* work, but I still need to test it out.

**Comment by Vaneixus** [comment]
I am currently planning better instructions with screenshots, but I am working on improving build times and functionality first. otherwise, I will be happy to include a much more detailed instructions if you need it. also, if you were using the RaspberryPi Imager tool, did you configure the host name and the user?

**Comment by nik1977** [comment]
Thank you for the reply.I have downloaded the image "fix typo", and burned it with Pi imager. First time i forgot to configure host  (i left it "raspberry" don't know if this is a problem) etc. but then reformatted the card and burned again the image .Result is the same.On Pi zero there is only one LED near camera connector.Think it's bicolor but it's blinking green .
I'll try the image from your link, hope it will work :)

**Comment by nik1977** [comment]
Still not working with Pi Zero ,but I tried with Pi4 and I see a new network .Still not working . Do I have to configure IP address ? Now it's set to automatic

**Comment by Vaneixus** [comment]
> Still not working with Pi Zero ,but I tried with Pi4 and I see a new network .Still not working . Do I have to configure IP address ? Now it's set to automatic

Well the network should be automatic. and did you try to unplug and replug the Pi Zero? I am currently testing this one: https://github.com/Vaneixus/rpi-gadget-image/actions/runs/6521120872

**Comment by Vaneixus** [comment]
if you want to try, you can manually set the ip address to 192.168.16.2

> Still not working with Pi Zero ,but I tried with Pi4 and I see a new network .Still not working . Do I have to configure IP address ? Now it's set to automatic

I just wanted to confirm, the Pi4 is currently able to automatically setup the network, or does it only show you the internet interface? it should show up as no connection, but if you type in hostname.local/vnc/, do you see anything?

**Comment by nik1977** [comment]
Yes i tried unplugging the Pi Zero but no effect.Now on Pi 4  it works, but may be too much current I get low voltage warning

**Comment by nik1977** [comment]
Now with ticker  cable rated 7 amps seems to be OK

**Comment by Vaneixus** [comment]
> Now with ticker  cable rated 7 amps seems to be OK

that is the Pi zero that started working, correct?

**Comment by Vaneixus** [comment]
you're getting access denied since you forgot to append /vnc/ or /code/ to
access the web applications.

**Comment by nik1977** [comment]
No  Pi Zero not working, but Pi4 is OK because I would like to use some  USB measurement devices with it.But  /code is not working which is not a problem for me.

**Comment by Vaneixus** [comment]
> No Pi Zero not working, but Pi4 is OK because I would like to use some USB measurement devices with it.But /code is not working which is not a problem for me.

I just wanted some reference, how much time do you wait for the connection to be established? the Pi Zero is a much weaker device, so I am making some assumption that it might be taking some time to be set up. if you're free to do it, can you connect it with a Data+Power cable for 5 minutes and then disconnect it then reconnect? if that does not work, can you try the latest image, I just finished testing it for boot up: https://github.com/Vaneixus/rpi-gadget-image/actions/runs/6521120872
oh, speaking of cables, is the cable you're using is a data + power cable? some cables are power only. could you tell me from where did you get it?

**Comment by Vaneixus** [comment]
I had another suggestion, but I need to know if you can interact with the pi via a keyboard,mouse and monitor. if yes, I would like that you login, and run the following command: ```sudo systemctl status multigadget``` then give me the output of the command. it should help in debugging this, sadly, I do not have a Pi Zero to test it, would have been much faster for me to assist.

**Comment by nik1977** [comment]
Hi,
I tried the latest image , waited long time but no success with Pi Zero.Cables I  use are Power and Data.SD card is 32GB could this be the problem?

**Comment by Vaneixus** [comment]
> Hi,
> I tried the latest image , waited long time but no success with Pi Zero.Cables I  use are Power and Data.SD card is 32GB could this be the problem?

SD card size isn't a problem. I am currently trying to figure out why it does not work. I will try to compare my code with a known pi zero config and maybe figure something out.

**Comment by Vaneixus** [comment]
> Hi,
> I tried the latest image , waited long time but no success with Pi Zero.Cables I  use are Power and Data.SD card is 32GB could this be the problem?

Hello, I just wanted to ask this, which port do you connect the USB cable to? is it the one labeled PWR or USB? if it's PWR try USB, otherwise try the other one.

**Comment by Vaneixus** [comment]
Hello again! I just tested the latest configuration on a Pi Zero 2, I just confirmed it that it needs to be plugged into the USB port labelled "USB" to get it to work. I tried it with the "PWR IN" port, and it did not work with that one, so, if you're willing to try it and haven't actually tried it yet, can you check if it works with the USB "USB" port?

**Comment by Vaneixus** [comment]
> Hi, I'm using pi zero w 1.1 with the prebuild image and nothing is happening only LED is blinking 7 times and pausing. I'm not sure if i have done things right.I have no experience with linux nor github. May be I'm missing some additional configurations.

Looking back at your comment, I only realised that you gave me the LED count, looking at https://forums.raspberrypi.com/viewtopic.php?t=348482 it seems to be an image write error. Can you tell me if this was still happening?

**Comment by nik1977** [comment]
Yes it still doesn't work.I used the same card on Pi4 and it's fine. Cabe is connected at port labeled USB, tried also PWR but it's only power lines I thing.

**Comment by Vaneixus** [comment]
Well on the RaspberryPi 4, the USB-C port is PWR by default, but it works due to the changes I make. So I wanted to see if that would be the case. but what's weird is that according to everything thus far, the card you use for RaspberryPi 4 works, but when connected to Pi Zero, you get a no boot partition found error(indicated by the 7 blinks). I will need to investigate this further, but I am hitting a brick wall, may... oh... I think I know why now... the images are all 64bit, the Pi Zero is 32Bits, I will create a special image and I need you to check it out.

**Comment by Vaneixus** [comment]
@nik1977 I will have a special image ready in 2 hours for you to download and try.

**Comment by Vaneixus** [comment]
My Apologies for the delay, I am having trouble with my working environment, I should be able to give you something within a few days. It requires some changes I have been planning for a while.

**Comment by nik1977** [comment]
Don't worry.I used to Pi4 and now i find Pi zero almost useless it's so slow.
